### DISCUSSION TITLE
Is there a good way to pass an auth token for a npm install-tool custom-registry override?

### DISCUSSION THREAD

**Discussion by pachuka** [containerbase/base#3437] [Q&A] [answered]
I followed https://github.com/containerbase/base/blob/main/docs/custom-registries.md for using our internal registry for npm tools, but our registry requires auth. Didn't see a good way to pass that through, so was wondering if there were any ideas around that.

Use case/context is in conjunction with the renovatebot github-action - https://github.com/renovatebot/github-action

**Comment by viceice** [comment, marked as answer]
I think your best option is to mount a `.npmrc` file to `/home/ubuntu/.npmrc` with your registry token

**Reply by pachuka** [reply]
I figured as much! Thanks for the response, just wanted to make sure I wasn't missing any sort of other override parameter :)
